lib/discordgo/events.go

```diff
@@ -561,4 +578,4 @@ type GuildAuditLogEntryCreate struct {
 type GuildJoinRequestUpdate struct{}
 type GuildJoinRequestDelete struct{}
 type VoiceChannelStatusUpdate struct{}
-type ChannelTopicUpdate struct{}
\ No newline at end of file
+type ChannelTopicUpdate struct{}
```

Comment:
here you add new struct to the fold, so it would be safer to execute that eventhandler script, comment in the beginning of the file recommends - or copy stuff from eventhandlers.go, manual - error prone.

otherwise looks good, if something was wrong, bot would crash at once due to SEGFAULT

Reply:
<img width="849" alt="Screenshot 2024-03-07 at 18 43 26" src="https://github.com/botlabs-gg/yagpdb/assets/110698921/ed87d7d7-68ef-43d7-b3d0-d8024df9484d">
I thought that was just a whitespace removal by `go fmt`, but I'm not totally sure. How could I tell that I added a function by accident?

Reply:
oh, cool, already there then... no problem, my bad : )